book.rb

```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = rentals
```

Comment:
- [ ] Hi, please notice that you are attempting to use a variable named `rentals` which does not exist. therefore your variable `@rentals` will be set to `nil`. you should simply set rentals to and empty array.

Reply:
Done